docs/restic.md

```diff
@@ -1,23 +1,33 @@
 # Restic Integration
 
-Ark now has support for backups/restores of pod volume data using [restic][1], an open-source tool for doing
-backups and restores of filesystem data. This enables you to take backups of additional Kubernetes volume 
-types beyond those with snapshot APIs that are integrated with Ark (i.e. AWS EBS, GCP PD, Azure PD). It also lays 
-the foundation for future work to support cross-cloud stateful migrations. Note that the details of this feature,
-including names, commands, etc., may change as we receive feedback and refine our implementation. 
-
-Two new Ark custom resources have been created to support this feature: `PodVolumeBackup` and `PodVolumeRestore`.
-Additionally, a new Ark daemonset has been created that runs two controllers, one for each of the two new CRDs, on
-each node in the cluster. When an Ark backup is created that includes pods annotated for restic backup, the main Ark 
-backup controller will create a `PodVolumeBackup` custom resource that's owned by the `Backup`. The pod volume backup
-controller running on the pod's node will observe the new custom resource, and will run a restic backup of the volume 
-(accessing the volume's data via a hostPath mount of `/var/lib/kubelet/pods`). The main Ark backup controller will 
-wait for the `PodVolumeBackup` to complete before completing the Ark backup. Restores proceed similarly with some 
-minor differences to account for the fact that a new pod/volume is being created.
+As of v0.9.0, Ark has support for backing up and restoring Kubernetes volumes using a free open-source backup tool called 
+[restic][1].
+
+Ark has always allowed you to take snapshots of persistent volumes as part of your backups, provided you’re using one of 
+the supported cloud providers’ block storage offerings (Amazon EBS Volumes, Azure Managed Disks, Google Persistent Disks). 
+And, since v0.6.0, we’ve had a plugin model which enables anyone to easily implement additional object and block storage 
+backends, outside of the main Ark repository.
+
+We integrated restic with Ark so that users have an out-of-the-box solution for backing up and restoring any type of Kubernetes
+volume*. We view this as a new capability for Ark, rather than a replacement for existing functionality. If you're running on AWS, and
```

Comment:
I would update the asterisk to have this: `[*](#limitations)`. That will let you link to the Limitations heading, and you can remove the bulletpoint.

This does, however, make the click target kind of small, so maybe it could be reworded.

How about:
`We integrated restic with Ark so that users have an out-of-the-box solution for backupg up and restoring any type of Kubernetes volume, including the [new local volume type][4], with some [limitations](#limitations)`

Reply:
https://gist.github.com/nrb/cec0367b7e5625538f0003ad35184b69 is a demonstration of the * as a link.

Reply:
d'oh, the bullet about hostPath not being supported was supposed to have the asterisk escaped so it actually displays as an asterisk.

Personally I think it's clearer to just have the footnote about hostPath there, as it's a specific exception to the statement about all volumes. I don't really want to highlight local volumes in the main text since that's not the point. @ncdc WDYT?

Reply:
That's fair. I like enabling the reader to jump directly to the limitations, since Markdown doesn't have great support for real footnotes.

With the asterisk escaped the formatting looks good. I'll defer to @ncdc and @Bradamant3 here.

Reply:
I'm fine with how it is currently, but I would say `restoring almost any type of Kubernetes volume` (add almost)